include/ocpp/v201/device_model.hpp

```diff
@@ -222,8 +209,8 @@ class DeviceModel {
     /// \param component_criteria
     /// \return
     std::vector<ReportData>
-    get_custom_report_data(const std::optional<std::vector<ComponentVariable>>& component_variables = std::nullopt,
-                           const std::optional<std::vector<ComponentCriterionEnum>>& component_criteria = std::nullopt);
+    get_report_data(const std::optional<std::vector<ComponentVariable>>& component_variables = std::nullopt,
```

Comment:
I preferred the `get_custom_report_data` name since it really makes clear what this function is for too.